frontend/pages/SoftwarePage/components/IconCell/IconCell.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+import { uniqueId } from "lodash";
+
+import Icon from "components/Icon";
+import { COLORS } from "styles/var/colors";
+import { IconNames } from "components/icons";
+
+const baseClass = "icon-cell";
+
+interface IIssueCellProps {
```

Comment:
Change this to IIconCellProps <3

Reply:
will do. gonna change this is the current branch I'm working on so you don't have to rereview.